floyd/client/files.py

```diff
@@ -6,70 +6,79 @@
 from floyd.log import logger as floyd_logger
 
 
-def get_files_in_directory(path, file_type):
+def get_unignored_file_paths(path, ignore_list=[], whitelist=[]):
     """
-    Gets the list of files in the directory and subdirectories
-    Respects .floydignore file if present
+    Given an ignore_list and a whitelist of glob patterns, returns the list of
+    file paths in a directory and its subdirectories that are not ignored.
     """
-    local_files = []
-    separator = os.path.sep
-    ignore_list = FloydIgnoreManager.get_list()
-
-    # make sure that subdirectories are also excluded
-    ignore_list_expanded = ignore_list + ["{}/**".format(item) for item in ignore_list]
```

Comment:
Yeah this wasn't a necessary and/or wasn't working. Then it became obsolete with [this commit](https://github.com/floydhub/floyd-cli/commit/262755904b1c7a4d99a7ab80ad1cb96fbf21f234), which took care of ignoring sub-directories.